test/sqlsrv/isPooled.php

```diff
@@ -0,0 +1,25 @@
+<?php
+include_once 'autonomous_setup.php';
+$conn1 = sqlsrv_connect( $serverName, $connectionInfo);
+$connId1 = ConnectionID($conn1);
+$conn1 = null;
+
```

Comment:
in fact, does it matter if you close first connection or not in your test?
- That's how you pool connections.